packages/palette/package.json

```diff
@@ -124,6 +124,7 @@
     "d3-interpolate": "^1.3.2",
     "d3-shape": "^1.3.5",
     "luxon": "^1.15",
+    "proportional-scale": "3.0.1",
```

Comment:
One of mine, simple enough: https://github.com/dzucconi/proportional-scale/blob/master/src/scale.ts